pkg/apis/config/controller/v1alpha1/types.go

```diff
@@ -43,152 +43,187 @@ type ControllerConfiguration struct {
 	// The maximum burst queries-per-second of requests sent to the Kubernetes apiserver
 	KubernetesAPIBurst *int32 `json:"kubernetesAPIBurst,omitempty"`
```

Comment:
Shall we maybe combine `KubernetesAPIBurst` and `KubernetesAPIQPS` (and maybe `KubeConfig` and `APIServerHost`) into a new struct?

Reply:
Ok, lets try that. That way, we stay consistent with this config file & the existing webhook config file.

Reply:
Looks like these are top-level options in the webhook component: https://github.com/cert-manager/cert-manager/blob/master/pkg/apis/config/webhook/v1alpha1/types.go#L37-L43

So, let's not put these options in a separate object.